src/Storefront/Resources/views/storefront/element/cms-element-form/form-types/newsletter-form.html.twig

```diff
@@ -2,42 +2,38 @@
     {% set formPrefix = 'cms-form-newsletter' %}
     <form action="{{ path(action) }}"
           method="post"
-          {# @deprecated tag:v6.7.0 - Form validation plugin will be replaced by form handler plugin. #}
-          {% if feature('ACCESSIBILITY_TWEAKS') %}data-form-handler="true"{% endif %}
-          {% if not feature('ACCESSIBILITY_TWEAKS') %}data-form-validation="true"{% endif %}
+          data-form-handler="true"
           id="{{ formPrefix }}"
           aria-label="{{ element.config.title.value ? element.config.title.value : 'newsletter.headline'|trans }}">
 
         <div class="form-content">
         <div class="row g-2">
             {% block cms_form_newsletter_select_action %}
-                {% if feature('ACCESSIBILITY_TWEAKS') %}
-                    {% set newsletterOptions %}
-                        <option selected="selected"
-                                value="{{ constant('Shopware\\Storefront\\Controller\\FormController::SUBSCRIBE') }}">
-                            {{ 'newsletter.subscribeOption'|trans }}
-                        </option>
+            {% set newsletterOptions %}
+                <option selected="selected"
+                        value="{{ constant('Shopware\\Storefront\\Controller\\FormController::SUBSCRIBE') }}">
+                    {{ 'newsletter.subscribeOption'|trans }}
+                </option>
 
-                        <option value="{{ constant('Shopware\\Storefront\\Controller\\FormController::UNSUBSCRIBE') }}">
-                            {{ 'newsletter.unsubscribeOption'|trans }}
-                        </option>
-                    {% endset %}
+                <option value="{{ constant('Shopware\\Storefront\\Controller\\FormController::UNSUBSCRIBE') }}">
+                    {{ 'newsletter.unsubscribeOption'|trans }}
+                </option>
+            {% endset %}
 
-                    {% sw_include '@Storefront/storefront/component/form/form-select.html.twig' with {
-                        label: 'newsletter.labelActionSelect'|trans|sw_sanitize,
-                        id: 'newsletterAction',
-                        name: 'option',
-                        options: newsletterOptions,
-                        validationRules: 'required',
-                        additionalClass: 'col-12',
-                        additionalSelectClass: 'contact-select',
-                        attributes: {
-                            'data-form-field-toggle': 'true',
-                            'data-form-field-toggle-target': '.js-field-toggle-newsletter-additional',
-                            'data-form-field-toggle-value': 'subscribe',
-                        },
-                    } %}
-                {% else %}
```

Comment:
The else case is gone, but the code is still there. Probably has to be removed as well.